solutions/ng-rx/src/app/services/todo.service.ts

```diff
@@ -0,0 +1,15 @@
+import { Todo } from './../store/todo.model';
+import { Injectable } from '@angular/core';
+import { Http } from '@angular/http';
+import { Observable } from 'rxjs/Observable';
+import 'rxjs/add/operator/map';
+
+@Injectable()
+export class TodoService {
+  constructor(private http: Http) { }
+
+  getTodos(): Observable<Todo[]> {
+    return this.http.get('http://localhost:3000/todos')
+      .map(res => res.json());
+  }
```

Comment:
I am not sure if getting the participants to setup a JSON server is a little excessive, since this exercise is about redux.  I wonder if there are any value to simply use Observable.just() to mock the http request.
`http://reactivex.io/documentation/operators/just.html#collapseRxJS`